examples/11_advanced_parametric_study.py

```diff
@@ -111,8 +111,8 @@ class provides data management features that make the work easier. Also, the
     layer_thicknesses=initial_layer_thicknesses,
     beam_diameters=initial_beam_diameters,
     heater_temperatures=initial_heater_temps,
-    min_area_energy_density=min_energy_density,
-    max_area_energy_density=max_energy_density,
+    min_pv_ratio=min_pv_ratio,
```

Comment:
@safeerehman If we stick with "pv_ratio", somewhere we need to say "v" stands for "velocity".